app/views/repositories/show.html.erb

```diff
@@ -11,16 +11,34 @@
 
 <div class="content-pane">
   <div id="repository-toolbar">
+    <div class="repository-title">
+      <span class="repository-label"><%= t('repositories.index.title_label') %> </span>
+      <span class="repository-name" title="<%= @repository.name %>"><%= @repository.name %></span>
+      <span class="repository-share-icon">
+        <% if @repository.shared_with?(current_team) %>
+          <% if @repository.team_repositories.first[:permission_level] == 'write' %>
```

Comment:
Wouldn't this check also be true for second shared team with `permission_level = read`? 

Additionally could we write this as:
```
   <% if @repository.team_repositories.first.permission_level.write? %>
```

Just a suggestion.

Reply:
Nice 1, thanks!